src/ui/buttons/buttons.jsx

```diff
@@ -189,6 +190,7 @@ export function Buttons(props : ButtonsProps) : ElementNode {
             }
 
             <Script
+                buttonAnimation={ createAddDivideLogoAnimation() }
```

Comment:
This is still not generic really. I think what you'd want is `createLogoAnimation()` here and have internal logic to decide which animation gets returned.